File: backend/drop_schemas.py
import motor.motor_asyncio
import asyncio

async def drop_schemas():
    client = motor.motor_asyncio.AsyncIOMotorClient("mongodb://localhost:27017")
    db = client["SSBC_DB"]
    
    # Drop all indexes on events collection
    await db.events.drop_indexes()
    print("Dropped all indexes on events collection")

if __name__ == "__main__":
    asyncio.run(drop_schemas())

File: backend/drop_schemas.js
// Switch to SSBC_DB database
db = db.getSiblingDB('SSBC_DB');

// Drop all indexes on events collection
db.events.dropIndexes();

// Create new index on ministry field
db.events.createIndex({ ministry: 1 }, { unique: true, name: "unique_ministry_index" });
